frontend/src/containers/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,63 @@
+.topLoginImage {
+	background-image: url("../../assets/Login_Page.jpg");
+	width:  100vw;
+	height: 60vh; 
+	background-size: cover;
+	background-position: center;
+	background-repeat: no-repeat;
+}
+
+.loginPage.container-fluid {
+    margin: 0px;
```

Comment:
Spaces vs. tabs! Here and further down.

Reply:
Not sure if what I did fixed this . . .Everything is now auto indented with tabs now, so it should be good